eventfiles/templates/file/event_files.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Event files</title>
+        {% load static %}
+    <link href="{% static 'event_files/event_files.css' %}" rel="stylesheet" type="text/css">
+     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="https://kit.fontawesome.com/ca9581fda8.js" crossorigin="anonymous"></script>
+  <script src="{% static 'event_files/event_files.js' %}"></script>
+    {% block head-content %}
+    {% endblock %}
+</head>
+{% block content %}
+<body>
+<center>
+<div class="container-shadow">
+     <div class="container">
+        <div class="wrap">
+            <h1 class="title" > {{title}} files:</h1>
+
+            <div class="file-list">
+                {% if files %}
+                    {% for post in files %}
+                <div class='PostContent'>
+                        <h2 class="FileName" >{{post}}</h2>
+                        <a href="download/?event={{event}}&file_name={{post}}"download="{{post.adminupload.url}}" class="DownloadButton">
+                             <i class="fa-solid fa-circle-down fa-2x" aria-hidden="true"></i>
+                        </a>
+                </div>
+                    {% endfor %}
+                {% endif %}
+            </div>
+
+            {% if not files %}
+                 <h2>No files exist</h2>
+            {% endif %}
+
+            <form action="./{{event}}" class="form" id="form1" method="POST" enctype="multipart/form-data">
+            <fieldset class="form-group">
+            {% csrf_token %}
+       <input id="uploadFile" placeholder="No file selected" disabled="disabled" />
+            <div class="fileUpload">
+                <span id="uploadBtn">Upload</span>
+                <input id="fileInput" type="file" name="{{form.file.name}}" class="upload" />
+            </div>
+
```

Comment:
since i have only one input given from the form which i made it hidden and made my own input, becuase i find it more easy to manipulate and handle then the one given by django. i find it unnecessary to also make it on the widgets